src/components/Link/Item.js

```diff
@@ -100,8 +100,8 @@ export default class LinkItem extends React.Component {
   };
 
   hoverHandlers = {
-    onTouchStart: this.toggle,
-    onMouseEnter: this.open,
+    onClick: this.toggle,
+    onMouseEnter: "ontouchstart" in document.documentElement ? null : this.open,
```

Comment:
This would cause server side rendering to fail

Reply:
Fixed.